eth_macd_strategy.py

```
@@ -0,0 +1,510 @@
+        side, score_state = determine_signal(indicators, inputs)
+    except Exception as exc:
+        logging.error("Indicator computation failed: %s", exc)
+        return
+
+    if side is None:
+        logging.info("No actionable signal. Scores: %s", score_state)
+        return
+
+    try:
+        balance = fetch_account_balance(user)
+    except Exception as exc:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)